pages/api/graphql-rest.page.ts

```diff
@@ -99,6 +132,35 @@ class MpdxRestApi extends RESTDataSource {
     }
   }
 
+  async createExportedContacts(
+    mailing: boolean,
+    format: string,
+    filter: {
+      account_list_id: string;
+      newsletter: string;
+      status: string;
+    },
+    labelType?: string | null,
+    sort?: string | null,
+  ) {
+    let pathAddition = '';
+    if (mailing) {
+      pathAddition = '/mailing';
+    }
+    const { data } = await this.post(`contacts/exports${pathAddition}`, {
+      data: {
+        params: {
+          filter,
+          type: labelType,
+          sort,
+        },
+        type: 'export_logs',
+      },
+    });
+
+    return `${process.env.REST_API_URL}contacts/exports${pathAddition}/${data.id}.${format}?access_token=`;
```

Comment:
Maybe let the frontend tack on the `access_token=`? Just seems a little odd to return part of a query param.